src/java/org/apache/cassandra/index/SecondaryIndexManager.java

```diff
@@ -382,6 +396,17 @@ public boolean isIndexQueryable(Index index)
         return isIndexQueryable(index.getIndexMetadata().name);
     }
 
+    /**
+     +     * Checks if the specified index is building.
+     +     *
+     +     * @param index the index
+     +     * @return <code>true</code> if the specified index is building, <code>false</code> otherwise
+     +     */
+    public boolean isIndexBuilding(Index index)
```

Comment:
Nit: can we put the two overloads of `isIndexBuilding` together?

Reply:
Soon...